valhalla/jawn/src/managers/score/ScoreManager.ts

```diff
@@ -60,4 +67,14 @@ export class ScoreManager extends BaseManager {
       return err(error.message);
     }
   }
+
+  private mapScores(scores: Scores): Score[] {
+    return Object.entries(scores).map(([key, value]) => {
+      return {
+        score_attribute_key: key,
+        score_attribute_type: typeof value === "boolean" ? "boolean" : "number",
+        score_attribute_value: typeof value === "boolean" ? (value ? 1 : 0) : value,
```

Comment:
Because we store score values as a BigInt by design and we can just map values, Idea is to make score aggregations in future and it will be much easier with BigInt